auslib/web/admin/swagger/api.yaml

```diff
@@ -1597,74 +1963,79 @@ definitions:
 
   DataVersionModel:
     title: Data Version
-    description: version number of object
+    description: "model to re-use 'data_version' field."
     type: object
-    required:
-      - data_version
     properties:
       data_version:
+        description: version number of object
         type: integer
         format: int32
         minimum: 1
         example: 3
 
-  ReleaseBase:
-    title: Release Base
-    description: "To Balrog, a 'release' is data about a related set of builds. This does _not_ match up with the concept of a 'release' being on the 'beta', 'release' or 'esr' channel elsewhere. In Balrog, each set of nightlies on any branch is considered a release"
+  SignoffsRequiredModel:
+    title: Signoffs Required
+    description: "model to re-use 'signoffs_required' field."
+    type: object
+    properties:
+      signoffs_required:
+        description: minimum number of signoffs required to execute the change
+        type: integer
+        format: int32
+        minimum: 1
+        example: 2
+
+  RoleModel:
+    title: User Role
+    description: "model to re-use 'role' field."
     type: object
     properties:
-      name:
-        description: Release name
+      role:
+        description: roles are used when signing off on Scheduled Changes
         type: string
-        maxLength: 100
-        example: "Firefox-47.0.2-build2-43.0.1"
+        minLength: 1
+        maxLength: 50
+        example: releng
 
+  ChannelModel:
+    title: Channel
+    description: "model to define 'channel' field in request body."
+    type: object
+    properties:
+      channel:
+        description: The update channel of the application request an update.
```

Comment:
that's the same definition of channel in the docs as above. What should I change this to ?